src/location/DAL/queries.ts

```diff
@@ -220,3 +220,34 @@ export const hierarchyQuery = (query: string, disableFuzziness: boolean): estype
     },
   },
 });
+
+export const searchByCoordinatesQuery = ({ lat, lon, limit, source }: GetGeotextSearchByCoordinatesParams): estypes.SearchRequest => {
+  const esQuery: estypes.SearchRequest = {
+    query: {
+      bool: {
+        filter: [
+          {
+            geo_shape: {
```

Comment:
Please add the `relation` parameter to the query and make it configurable, with the default set to `intersects`

Reply:
done